modelplugin/aether-5.0.0/modelmain.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2021-present Open Networking Foundation.
```

Comment:
I'm pretty sure this file is not needed anymore - see `testdevice-1.0.0` and `testdevice-2.0.0` models - they do not need this file, as it's generated on the fly as https://github.com/onosproject/onos-config-model/blob/master/pkg/model/plugin/compiler/templates/main.go.tpl